strutil/strutil.go

```diff
@@ -127,3 +127,27 @@ func SortedListContains(list []string, str string) bool {
 	}
 	return list[i] == str
 }
+
+// TruncateOutput truncates input data by maxLines, imposing maxBytes limit (total) for them.
+// The maxLines may be 0 to avoid the constraint on number of lines.
```

Comment:
The edge case of maxLines == 0 seems strange here. There's just no reason to call this function if all we want is to slice a buffer. That's literally data[len(data)-n:].

Reply:
You're right, removed, thanks.